geomstats/geometry/hypersphere.py

```diff
@@ -86,6 +95,48 @@ def to_tangent(self, vector, base_point):
 
         return tangent_vec
 
+    @staticmethod
+    def angle_to_extrinsic(point_angle):
+        """Convert point from angle to extrinsic coordinates.
+
+        Convert from the angle in radians to the extrinsic coordinates in
+        2d plane. Angle 0 corresponds to point [1., 0.] and is expected in [-Pi, Pi).
+        This method is only implemented in dimension 1.
+
+        Parameters
+        ----------
+        point_angle : array-like, shape=[...]
+            Point on the circle, i.e. an angle in radians in [-Pi, Pi].
+
+        Returns
+        -------
+        point_extrinsic : array_like, shape=[..., 2]
+            Point on the sphere, in extrinsic coordinates in Euclidean space.
+        """
+        cos = gs.cos(point_angle)
+        sin = gs.sin(point_angle)
+        return gs.stack([cos, sin], axis=-1)
+
+    @staticmethod
+    def extrinsic_to_angle(point_extrinsic):
+        """Compute the angle of a point in the plane.
+
+        Convert from the extrinsic coordinates in the 2d plane to angle in
+        radians. Angle 0 corresponds to point [1., 0.]. This method is only
+        implemented in dimension 1.
+
+        Parameters
+        ----------
+        point_extrinsic : array-like, shape=[...]
+            Point on the circle, in extrinsic coordinates in Euclidean space.
+
+        Returns
+        -------
+        point_angle : array_like, shape=[..., 2]
+            Point on the circle, i.e. an angle in radians in [-Pi, Pi].
+        """
```

Comment:
Check that self.dim == 1 and returns an explanatory error otherwise.

Reply:
same comment